docs/get-started/totally-looks-like/index.md

```diff
@@ -0,0 +1,206 @@
+# Finetuning ResNet50 on Totally Looks Like Dataset
+
+In this tutorial, we will use Finetuner to finetune embeddings on [Totally Looks Like dataset](https://sites.google.com/view/totally-looks-like-dataset) (TLL) dataset.
+The dataset consists of 6016 pairs of images (12032 in total).
+
+>Totally-Looks-Like is a dataset and benchmark challenging machine-learned representations to reproduce human perception of image similarity.
+
+Finetuner adopts the idea of `transfer learning` and `metric learning`. The rationale is
+1. TLL dataset is a relatively small dataset, it's not reasonable to train a large network, such as ResNet on TLL from scratch. So we train part of the network by freezing layers.
+2. TLL dataset consists of pairs of images that can be formed as a positive pair (same classes), and a random image can be considered as a negative pair. We can form a `triplet` and use the Finetuner `TripletLoss`. We expect after fine-tuning, the distance between positive pairs can be pulled closer, while the distance between positive and negative pairs can be pushed away.
+
+## Environment & Data Preparation
+
+We will download `left.zip` and `right.zip`, as stated before,
+each of them consists of 6016 images which can be formed into pairs based on the same file name.
+
+
+```shell
+pip install gdown
+pip install finetuner
+pip install torchvision
```

Comment:
```suggestion
pip install torchvision
pip install torch>=1.9.0
```

Reply:
i think the latest torchvision will take care the torch version so I did not add it

Reply:
I see. The reason is due to my `pip` is not up-to-date